crates/ruff_linter/src/rules/pyupgrade/snapshots/ruff_linter__rules__pyupgrade__tests__UP050.py.snap

```diff
@@ -0,0 +1,322 @@
+---
+source: crates/ruff_linter/src/rules/pyupgrade/mod.rs
+---
+UP050.py:23:12: UP050 [*] Class with type parameter list inherits from `Generic`
+   |
+21 | ### Errors
+22 |
+23 | class C[T](Generic[_A]): ...
+   |            ^^^^^^^^^^^ UP050
+24 | class C[T](Generic[_B], str): ...
+25 | class C[T](int, Generic[_C]): ...
+   |
+   = help: Convert to new-style
+
+ℹ Unsafe fix
+20 20 | 
+21 21 | ### Errors
+22 22 | 
+23    |-class C[T](Generic[_A]): ...
+   23 |+class C[T, _A]: ...
+24 24 | class C[T](Generic[_B], str): ...
+25 25 | class C[T](int, Generic[_C]): ...
+26 26 | class C[T](bytes, Generic[_D], bool): ...
+
+UP050.py:24:12: UP050 [*] Class with type parameter list inherits from `Generic`
+   |
+23 | class C[T](Generic[_A]): ...
+24 | class C[T](Generic[_B], str): ...
+   |            ^^^^^^^^^^^ UP050
+25 | class C[T](int, Generic[_C]): ...
+26 | class C[T](bytes, Generic[_D], bool): ...
+   |
+   = help: Convert to new-style
+
+ℹ Unsafe fix
+21 21 | ### Errors
+22 22 | 
+23 23 | class C[T](Generic[_A]): ...
+24    |-class C[T](Generic[_B], str): ...
+   24 |+class C[T, _B: int](str): ...
```

Comment:
there's actually two reasons why the original code would have raised an exception:
- It mixes a type parameter list with explicit inheritance from `Generic`
- Even if it didn't have a type parameter list, it would have raised an exception because `Generic` must be the last base class in the bases tuple

I think this is okay, however, since the rule is concerned with correctness and the fix is clearly documented to always change runtime behaviour.